### PR TITLE
Tidying up anaconda-notebook repo

### PR DESCRIPTION
**Pull request by ijstokes**
README, version, license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,90 @@
...
+we're aiming to get rid of `anaconda-notebook` altogether).
+
+## Feedback, Bugs, Support
+
+We love feedback.  Please open an issue on this repo to give any feedback or
+open a PR if you have some code-change suggestions.
+
+Found a bug? Open an issue.
+
+Support? The (Anaconda mailing list)[https://groups.google.com/a/continuum.io/forum/#!forum/anaconda]
```
Comment (malev) [inline README.md]: the link should be:
[Anaconda mailing list](https://groups.google.com/a/continuum.io/forum/#!forum/anaconda)

File: README.md [hunk 2]
```diff
@@ -1,24 +1,90 @@
...
+of tools and custom configuration of Jupyter (in the normal way) that make it
+work in a way that will be beneficial for some users.  You still need regular
+Jupyter installed.
+
+**NOTE**: This "bundle" is designed to encapsulate a set of extensions into a
+`conda` environment.  Because Jupyter currently will only look in one place
+for extensions the `anaconda-notebook` wrapper points Jupyter to a directory
+specific to the conda environment to pick up the set of conda-installable
+extensions.  An implication of this is that **all other normally installed
+Jupyter extesions in `~/.jupyter` will not be available*.  Yes, this is
```
Comment (malev) [inline README.md]: missing `*` to close `**`

File: README.md [hunk 3]
```diff
@@ -1,24 +1,90 @@
...
+pieces are:
+
+* `anaconda-notebook` which is a meta-package specifying a set of sub-packages
+as a cohesive bundle.
+
+* `nbwrapper` which provides some "common code" and the actual wrapper scripts
+(yes, confusingly it is `nbwrapper` that includes the `anaconda-notebook` **script**,
+not the `anaconda-notebook` conda package!)
+
+`nbwrapper` has its own `README` file (here)[nbwrapper/README.md]
```
Comment (malev) [inline README.md]: link needs to be fixed.

**Comment by mmarchetti** [comment]
We should remove the nbwrapper build from .binstart.yml. 

anaconda.org is looking in the nbwrapper directory for .binstar.yml (and there is one there too).

File: README.md (hunk 1 above)
Comment (ijstokes) [inline README.md]: Corrected, thanks.

**Comment by ijstokes** [comment]
@malev and @mmarchetti if you have a moment, can you let me know if this looks OK now?

**Comment by mmarchetti** [comment]
Looks good. My only comment is that you might want to comment out win-32 and win-64 so the CI checks can pass.

**Comment by ijstokes** [comment]
win-64 with Python 2.7  is also failing.  I'm going to leave that line in for now on the basis that it should be there and the failure is external to us (TTBOMK).

**Pull request merged**